openedx/core/djangoapps/organization_api/api/views.py

```diff
@@ -0,0 +1,39 @@
+"""
+Organizations API views.
+"""
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.status import HTTP_404_NOT_FOUND
+from rest_framework.views import APIView
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from openedx.core.lib.api.authentication import JwtAuthentication
+from util.organizations_helpers import get_organization_by_short_name
+
+
+class OrganizationsView(APIView):
```

Comment:
This is not a model view. I just a simple API view which uses and existing organizations helper method `get_organization_by_short_name` to return data.